Source/Public/Connect-ADOPS.ps1

```diff
@@ -8,7 +8,6 @@ function Connect-ADOPS {
         
         [Parameter(ParameterSetName = 'Interactive')]
         [Parameter(ParameterSetName = 'ManagedIdentity')]
-        [Parameter(ParameterSetName = 'OAuthToken')]
```

Comment:
While I agree this is a good fix, could we please revert it and make it a bug for fixing in future version -> 3? Since it would be a breaking change (_if_ someone by chance happen to have it in a script somewhere..)

Reply:
Sounds like a plan =) I'll revert it and we can put it on the list for a upcoming major release.

Reply:
Perfect-o! Thnks!